app/src/Containers/UserForm/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useRef, useState } from "react";
+import { passwordValidation } from "../../Helpers/password-validation";
+import { UserFormWrapper } from "./styles";
+import qs from "qs"
+
+interface user {
+  username: string,
+  password: string,
+}
+
+const reqUserCreate = async (newUser: user) => {
+  const in_flight = await fetch("/api/user/create", {
+    method: 'POST',
+    body: JSON.stringify(newUser),
+    headers: {
+      'Content-Type': "application/json"
+    }
+  });
+  const output = await in_flight.json();
+  return output
+}
+
+const reqLogin = async (user: user) => {
+  const in_flight = await fetch('/api/user/login', {
+    method: 'POST',
+    body: JSON.stringify(user),
+    headers: {
+      'Content-type': 'application/json'
+    }
+  })
+  const output = await in_flight.json();
+  return output
+}
+
+const creationErrorHandler = (res: any) => {
+  if(res.code === 11000){
+    alert("That username is already in use.")
+    return false
+  }
+}
+
+const loginErrorHandler = (res: any) => {
+}
+
+const UserForm = (props: any) => {
+  
+  const [newAccount, setNewAccount] = useState(false)
+  const usernameInput = useRef()
+  const passwordInput = useRef()
+  const passwordCheckInput = useRef()
+  const [errorMessage,setError] = useState('')
+
+  const handleCreate = (userInfo: user, passwordCheck: string) => {
```

Comment:
This component is a couple deep but it is where the token actually gets set. And I know its possible to share this with parent components.